server/chat-plugins/friends.ts

```diff
@@ -0,0 +1,623 @@
+/**
+ * Friends list plugin.
+ * Allows for adding and removing friends, as well as seeing their activity.
+ * Written by Mia-pi.
+ * @author mia-pi-git
+ */
+import * as Sqlite from 'better-sqlite3';
+import {Utils} from '../../lib/utils';
+import {FS} from '../../lib/fs';
+
+/** Max friends per user */
+const MAX_FRIENDS = 50;
```

Comment:
Seems low, if you're worried about DOS-ability I'd bump the limit for voices+ at least